### DISCUSSION TITLE
How to Signout only current session?

### DISCUSSION THREAD

**Discussion by kienns1996** [supabase/gotrue-csharp#86] [Q&A]
SignOut() function of Gotrue-csharp log out all sessions by default, is it possible to SignOut() with scope option "global", "local" or "others"?

**Comment by corcorpj** [comment]
I know this is extremely late - but the StatelessClient implementation will support Signout provided a 'jwt' which in essence targets a specific Session. 

However, fair warning that my understanding of this sdk is flawed, and have lots of questions about the complexity of this abstraction and its function in server side applications.
